### DISCUSSION TITLE
Suggestion: Integrate LitVar 2.0 API call on variant pages

### DISCUSSION THREAD

**Discussion by ehre** [Clinical-Genomics/scout#5292] [General]
It might be useful to integrate LitVar 2.0 API calls on the SNV/INDELs variant pages, https://www.ncbi.nlm.nih.gov/research/litvar2/api

While it appears that LitVar is still less comprehensive than HGMD or Genomenon MasterMind, LitVar is fully open source and an official NCBI project that connects to other relevant databases. Furthermore, with version 2.0 the number of searchable published variants have increased from just under 2 million to almost 14 million variants ( https://pubmed.ncbi.nlm.nih.gov/37268776/ ).

**Comment by dnil** [comment]
Unfortunately they seem to use their own variant id system, so annotating with them would make the most sense. But, there some more complicated options available from the API. First, their API would allow a two stage process of searching for matches to dbSNP ids, which would then return a link to be added. This could be automated fully, so it's the one I would suggest we go for for now. 

Then there is a "free text" type search (with the `autocomplete` API endpoint) which can result in multiple hits, and variant free text names in their db is mixed between dbSNP ids, a fixed gene symbol plus protein change, gene symbol plus DNA change and genomic coordinates, sometimes intervals for SVs. There are not a ton of references linked to these more eh generic descriptors, but slightly more than to make them entirely negligible. This is a larger issue, perhaps involving some manual input or a cascade of requests and rules to pick the "right" variant from the autocomplete results. Honestly, I believe they will have to try to establish some more universal variant descriptor in a later version.

**Comment by dnil** [comment]
Hi, I saw a somewhat similar system in the last months [VEP release](https://grch37.ensembl.org/info/docs/tools/vep/script/vep_download.html#new) called [AVADA](https://pubmed.ncbi.nlm.nih.gov/31467448/). The publication included some of the HGMD authors. While it may or may not be true anymore that they are most complete today, at least it should be very easy to integrate in pipelines and Scout.

**Comment by ehre** [comment]
Sounds interesting. I guess the challenge with evaluating different systems is benchmarking against a truth set, and also trying to understand how "official" and/or long term a specific project is. I don't know if AVADA has been updated since the 2020 publication, and I don't know if it is supported by any larger organization? But I guess the inclusion in VEP holds some promise.

**Comment by dnil** [comment]
I don't have any inside info on this. Some further digging seems to indicate the new plugin is using the grch37 results from the 2016 literature dump, which cools my initial enthusiasm considerably. I guess let's keep monitoring. So far, no changes on the LitVar2 API/download side.

**Comment by dnil** [comment]
VEP also has a plugin for [Mastermind](https://github.com/Ensembl/VEP_plugins/blob/release/112/Mastermind.pm), the Genomenon effort, which has continuously updated to public downloadable data releases (https://www.genomenon.com/cvr/), with variant ids linking back to their site. The site has a free "fair use" policy, but quite quickly will stop showing hits and asking for a PRO license. If we have users that anyway use them this might be acceptable.

**Comment by ehre** [comment]
I think a Mastermind integration could be useful.
